src/datePicker/Day.tsx

```diff
@@ -57,7 +69,8 @@ export const Day: React.FC<Props> = ({
     const date = getDayFormat(day, jalali);
 
     if (
-      (disabledBeforeToday && dayjs(day).isBefore(dayjs())) ||
+      (disabledBeforeDate && dayjs(day).isBefore(disabledBeforeDate)) ||
```

Comment:
Also for this part: 

`  (disabledBeforeDate && dayjs(dayjs(date)).isBefore(disabledBeforeDate)) ||
      (disabledAfterDate && dayjs(dayjs(date)).isAfter(disabledAfterDate))`